src/lib/formValidator.js

```diff
@@ -0,0 +1,103 @@
+import { monthFullNames } from '../modules/Experiences/AddOrEditExperienceForm/staticData';
+
+const experienceValidator = (values) => {
+  let isError = false;
+  let err = {};
+
+  if (values.host.length < 1) {
```

Comment:
These length validations on 'host' and 'title' will throw an exception if they get a null or undefined value, it may make sense to handle that as an explicit error instead

Reply:
I changed to --> if (!values.title) and if (!values.start_month).  Is that what you meant?

Reply:
I assume you changed `host` as well? That should work

Reply:
Yes (!values.host)